proposals/gc/MVP.md

```diff
@@ -289,36 +304,38 @@ Perhaps also the following short-hands:
 
 #### Runtime Types
 
-* `rtt.canon <constype>` returns the RTT of the specified type
-  - `rtt.canon t : [] -> [(rtt t)]`
+* `rtt.canon <heaptype>` returns the RTT of the specified type
+  - `rtt.canon t : [] -> [(rtt 0 t)]`
   - multiple invocations of this instruction yield the same observable RTTs
   - this is a *constant instruction*
   - equivalent to `(rtt.sub t (rtt.canon any))`, except when `t` itself is `any`
```

Comment:
What is the type of `(rtt.canon any)`? Does n go negative, or is the typing of `rtt.sub` different for any?

Reply:
Ah, good catch! Wrt handling `any` this was off by one. Fixed rule and description to something more sensible.

Reply:
Ah yes, fixed (it's `(rtt.sub 1 any t (rtt.canon any))`).